runtime/moonbase/src/lib.rs

```diff
@@ -1216,9 +1216,9 @@ impl_runtime_apis! {
 			let params = (&config, &whitelist);
 
 			add_benchmark!(params, batches, parachain_staking, ParachainStakingBench::<Runtime>);
-			add_benchmark!(
-				params, batches, pallet_crowdloan_rewards, PalletCrowdloanRewardsBench::<Runtime>
-			);
+			// add_benchmark!(
+			// 	params, batches, pallet_crowdloan_rewards, PalletCrowdloanRewardsBench::<Runtime>
+			// );
```

Comment:
@girazoki I see there is a benchmarking branch in the crowdloan repo, but the dependency in these runtimes does not point to it

Here is the error when I uncomment the lines commented out above:

```
  error[E0599]: no function or associated item named `benchmarks` found for struct `pallet_crowdloan_rewards::Pallet<Runtime>` in the current scope
      --> /Users/4meta5/ong/ps/moonbeam-chain/runtime/moonriver/src/lib.rs:1263:4
       |
  1263 | /             add_benchmark!(
  1264 | |                 params, batches, pallet_crowdloan_rewards, PalletCrowdloanRewardsBench::<Runtime>
  1265 | |             );
       | |______________^ function or associated item not found in `pallet_crowdloan_rewards::Pallet<Runtime>`
```

Reply:
I motion to keep the commented code, and uncomment it in a followup when the crowdloan pallet is ready for it.